CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 9.10.1
```

Comment:
```suggestion
# 9.11.0
```

This isn't a bug fix, so should be a minor version update. As per: https://semver.org/